runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -20,8 +20,61 @@
+   // TODO:ARM64: Enable when Implemented: tet[TR::instanceof] = TR::TreeEvaluator::instanceofEvaluator;
+   tet[TR::checkcast] = TR::TreeEvaluator::checkcastEvaluator;
+   tet[TR::checkcastAndNULLCHK] = TR::TreeEvaluator::checkcastAndNULLCHKEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::New] = TR::TreeEvaluator::newObjectEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::variableNew] = TR::TreeEvaluator::newObjectEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::newarray] = TR::TreeEvaluator::newArrayEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::anewarray] = TR::TreeEvaluator::anewArrayEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::variableNewArray] = TR::TreeEvaluator::anewArrayEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::multianewarray] = TR::TreeEvaluator::multianewArrayEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::arraylength] = TR::TreeEvaluator::arraylengthEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::ResolveCHK] = TR::TreeEvaluator::resolveCHKEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::DIVCHK] = TR::TreeEvaluator::DIVCHKEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::BNDCHK] = TR::TreeEvaluator::BNDCHKEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::ArrayCopyBNDCHK] = TR::TreeEvaluator::ArrayCopyBNDCHKEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::BNDCHKwithSpineCHK] = TR::TreeEvaluator::BNDCHKwithSpineCHKEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::SpineCHK] = TR::TreeEvaluator::BNDCHKwithSpineCHKEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::ArrayStoreCHK] = TR::TreeEvaluator::ArrayStoreCHKEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::ArrayCHK] = TR::TreeEvaluator::ArrayCHKEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::MethodEnterHook] = TR::TreeEvaluator::conditionalHelperEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::MethodExitHook] = TR::TreeEvaluator::conditionalHelperEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::allocationFence] = TR::TreeEvaluator::flushEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::loadFence] = TR::TreeEvaluator::flushEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::storeFence] = TR::TreeEvaluator::flushEvaluator;
+   // TODO:ARM64: Enable when Implemented: tet[TR::fullFence] = TR::TreeEvaluator::flushEvaluator;
+
+   tet[TR::icall]  = TR::TreeEvaluator::directCallEvaluator;
+   tet[TR::lcall]  = TR::TreeEvaluator::directCallEvaluator;
+   tet[TR::fcall]  = TR::TreeEvaluator::directCallEvaluator;
+   tet[TR::dcall]  = TR::TreeEvaluator::directCallEvaluator;
+   tet[TR::acall]  = TR::TreeEvaluator::directCallEvaluator;
+   tet[TR::call]   = TR::TreeEvaluator::directCallEvaluator;
+   tet[TR::vcall]  = TR::TreeEvaluator::directCallEvaluator;
```

Comment:
Why are these being initialized here?  This is the default value that the OMR code generator initializes them to.

Reply:
Not sure Just following the lead of the P compiler for this one.